test/test_core.py

```diff
@@ -8651,7 +8661,7 @@ def test_memprof_requirements(self):
           if (typeof _emscripten_stack_get_base === 'function' &&
               typeof _emscripten_stack_get_end === 'function' &&
               typeof _emscripten_stack_get_current === 'function' &&
-              typeof Module['___heap_base'] === 'number') {
+              Module['___heap_base'] > 0) {
```

Comment:
Hmm.. so the type of `Module['___heap_base']` in DECLARE_ASM_MODULE_EXPORTS=0 is no longer a number and is now a wasm global?    Does > 0 work on wasm globals?   Shouldn't we export the `.value` of the global instead?

Reply:
In that case I'm includes to say we should skip this test then since it depends on `Module['___heap_base']` being a number.

And we should open a bug to make `DECLARE_ASM_MODULE_EXPORTS=0` unwrap wasm globals like `DECLARE_ASM_MODULE_EXPORTS=1` does.

Reply:
I guess since this test is specifically about the memory profiler it doesn't matter, but maybe open the bug anyway since the behaviour seems wrong/inconsistent.

I'm supprised we don't have other tests that confirm that exported wasm globals are actually numbers?  We probably should if we don't already.